fusor-ember-cli/app/templates/components/deployment-role.hbs

```diff
@@ -11,7 +11,7 @@
         {{role.name}}
     </div>
     {{#if roleAssigned}}
-         <select class="selectpicker"  {{action "updateNodeCount" this on="change"}}>
```

Comment:
This class `selectpicker` was causing `display: none;` in development and it seemed to be coming from patternfly css. Very strange. I searched and don't see any code that uses css class `selectpicker`, so I'm renaming to see select boxes in development setup.